### DISCUSSION TITLE
How can I join the eXo Tribe?

### DISCUSSION THREAD

**Discussion by sergeByishimo** [exoplatform/exo-community#21] [Q&A]
Hi,

I want to contribute by fixing bugs I encouter and developing features. How can I join the eXo Tribe?

**Comment by rdenarie** [comment]
Hello.

eXo Tribe is now internal tool, and it is not possible to register on it.

But you can contribute by proposing PR on different eXo modules. It will be reviewed by our technical architects and merged is possible

**Comment by sergeByishimo** [comment]
Hi!
How can I become part of the exo internal team?

**Comment by rdenarie** [comment]
I mean, it is for employee only.

We have plan to reopen it to the community. Stay tuned !

**Reply by sergeByishimo** [reply]
Alright. Thanks.
